release-notes/unreleased/LUN-383-fav-icon-displaced.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: LUN-383
+show_in_stores: false
+platforms:
+  - android
+  - ios
+de: Das Favoriten Icon in Übungen wurde korrekt positioniert.
```

Comment:
(optional) I like when release notes sound more optimistic and less "buggy". E.g. "Die Positionierung des Icons wurde optimiert".